stdlib/SparseArrays/test/sparse.jl

```diff
@@ -82,6 +82,8 @@ end
         SparseMatrixCSC(6, 6, big.([1,2,4,7,8,9,9]), big.([1,1,2,1,2,3,4,5]), big.([1,2,3,4,5,6,7,8])))
     @test sparse(Any[1,2,3], Any[1,2,3], Any[1,1,1]) == sparse([1,2,3], [1,2,3], [1,1,1])
     @test sparse(Any[1,2,3], Any[1,2,3], Any[1,1,1], 5, 4) == sparse([1,2,3], [1,2,3], [1,1,1], 5, 4)
+    # with combine
+    @test sparse([1, 1, 2, 2, 2], [1, 2, 1, 2, 2], 1.0, 2, 2, +) == sparse([1, 1, 2, 2], [1, 2, 1, 2], [1.0, 1.0, 1.0, 2.0], 2, 2)
```

Comment:
That's okay, but it feels more like testing whether `+` is the default. How about testing a non-default operation?
```julia
@test sparse([1, 1, 2, 2, 2], [1, 2, 1, 2, 2], 2, 2, 2, *) == sparse([1, 1, 2, 2], [1, 2, 1, 2], [2, 2, 2, 4], 2, 2)
```

Reply:
I can add that!

Reply:
Or something similar with `-1`...

Reply:
Yeah, anything that would take an independent path from the default `combine=+` operation.